### PR TITLE
Added Azure arc documentation

### PR DESCRIPTION
**Pull request by rahul188**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+This documentation provides step-by-step instructions on how to install the New Relic Extension on an Azure Arc Connected Machine using the Azure Command-Line Interface (Azure CLI).
+
+## Prerequisites
+
+Before you begin, ensure that you have:
+
+- Azure CLI installed on your local machine.
+- Azure Arc Connected Machine set up and registered in your Azure subscription.
+- The necessary permissions to manage resources in the specified resource group.
```
Comment (akristen) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx:19]: Does this vary by user? Can we give them guidance on those permissions?

File: src/nav/infrastructure.yml [hunk 2]
```diff
@@ -734,6 +734,8 @@ pages:
     pages:
       - title: Introduction to Azure integrations 
         path: /docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations
+      - title: Azure Arc New Relic
```
Comment (akristen) [inline src/nav/infrastructure.yml:737]: We don't typically label categories this way, even though I see it looks like a pattern in the other category. Are there alternative names we can use? For example, maybe 'Set up New Relic with Azure Arc'?

**Comment by akristen** [comment]
I see this is still a draft and you don't want to merge it yet. When you're ready for a tech writer review, reach out to our hero in the channel and we'll get this reviewed and merged. In the meantime, it might be good to weave in more context, especially in the intro.

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx (hunk 1 above)
Comment (rahul188) [reply src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx:31]: No, it required same permission which user had while creating the resource.

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+title: Install the New Relic Extension On Azure Arc
+tags:
+  - Infrastructure
+  - Install NewRelic Extension
+  - NewRelic Extension
+  - Azure Arc Extension
+metaDescription: New Relic's Extension For Azure Arc.
+---
+
+This documentation provides step-by-step instructions on how to install the New Relic Extension on an Azure Arc Connected Machine using the Azure Command-Line Interface (Azure CLI).
```
[the remark this answers is not in the document]
Comment (rahul188) [reply src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx]: we have a similar extension for Azure VMSS as well.
Doc - https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/azure-extensions-infrastructure/

File: src/nav/infrastructure.yml (hunk 2 above)
Comment (rahul188) [reply src/nav/infrastructure.yml:737]: @akristen, I made suggested change...

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx]: @rahul188 this is a similar extension but not the same as the one the doc is referring to?

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx:31]: @rahul188 can you be more specific about this? what type of permissions are those?

Comment (rahul188) [reply src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx:31]: @homelessbirds , it should be part of Azure Subscription Owner or Contributor Role.

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (rahul188) [reply src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-arc-integration.mdx]: @homelessbirds, currently there is no Azure Arc doc in NR and this is (will be) the initial version of the document.

**Review by jeff-colucci — APPROVED** [review]

**Review by jeff-colucci — APPROVED** [review]

**Pull request merged**